manifests/service.pp

```diff
@@ -0,0 +1,40 @@
+# Class: datadog_agent::service
+#
+# This class declares the datadog-agent service
+#
+
+class datadog_agent::service(
+  $service_ensure = 'running',
```

Comment:
```suggestion
  String $service_ensure = 'running',
```

Reply:
This can also be an enum type (ie: `running` without quotes) so I don't want to force the type.